crates/holochain_zome_types/src/block.rs

```diff
@@ -22,7 +22,7 @@ pub struct BlockAgentInput {
 }
 
 /// Reason why we might want to block a cell.
-#[derive(Clone, serde::Serialize, serde::Deserialize, Debug)]
+#[derive(Clone, serde::Serialize, serde::Deserialize, Debug, Eq, PartialEq)]
```

Comment:
Added for testing. I'm happy with this always being available but does anybody feel strongly that this should be behind a cfg_attr?

Reply:
always is fine by me